src/configures/YnETHxConfigurer.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: BSD Clause-3
+pragma solidity ^0.8.24;
+
+import {MainnetActors, IActors} from "script/Actors.sol";
+import {MainnetContracts as MC} from "script/Contracts.sol";
+import {YnETHxVault} from "src/YnETHxVault.sol";
+import {BaseRules} from "script/rules/BaseRules.sol";
+import {ConnectorRules} from "script/rules/ConnectorRules.sol";
+import {YieldNestRules} from "script/rules/YieldNestRules.sol";
+import {BaseRoles} from "script/roles/BaseRoles.sol";
+
+contract YnETHxConfigurer is MainnetActors {
+    error NotAdmin();
+    error InvalidVaultVersion();
+
+    function configure(address provider, address withdrawer) external {
```

Comment:
what's the gas consumption of running this? as reported by --gas-report

Reply:
concerned about whether we need to make this an iterative run if gas is ridiculously high

which will complicate it so it's better to measure asap

Reply:
```bash
FOUNDRY_PROFILE=mainnet forge test --gas-report --match-test test_configure
```

<img width="994" alt="Screenshot 2025-02-05 at 2 15 04 PM" src="https://github.com/user-attachments/assets/238a6daa-8b7f-4a69-b192-8e0c6ecbbe90" />

Reply:
good thanks!